src/huggingface_hub/file_download.py

```diff
@@ -1140,7 +1139,6 @@ def hf_hub_download(
         blob_path = "\\\\?\\" + os.path.abspath(blob_path)
 
     with FileLock(lock_path):
-
```

Comment:
late to the party, but please, no whitespace only diffs if possible 😇

Reply:
I thought this was some style change due to black, it was failing if not :D